Makefile

```diff
@@ -24,9 +24,8 @@ DEIS_REGISTRY ?= ${DEV_REGISTRY}
 
 IMAGE_PREFIX ?= deis/
 
-# Kubernetes-specific information for RC, Service, and Image.
-RC := manifests/deis-${SHORT_NAME}-rc.yaml
-SVC := manifests/deis-${SHORT_NAME}-service.yaml
+# The canonical name for the Docker image produced at the end of the
+# build cycle.
 IMAGE := ${DEIS_REGISTRY}/${IMAGE_PREFIX}${SHORT_NAME}:${VERSION}
```

Comment:
we've been talking about omitting the trailing `/` from `${DEIS_REGISTRY}` so that we can push to dockerhub. seems like that's the consensus as of now